docs/samples/contact-center/app.js

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-env browser */
+
+/* global Webex */
+/* global Calling */
```

Comment:
Do we need this?

Reply:
Webex is needed, might not need Calling. Removed it